dotCMS/src/main/java/com/dotcms/ai/api/EmbeddingsAPI.java

```diff
@@ -140,15 +140,31 @@ static EmbeddingsAPI impl() {
     void initEmbeddingsTable();
 
     /**
-     * this method takes a snippet of content and will try to see if we have already generated embeddings for it.
-     * It checks the cache first, and returns if it finds it there.  Then it checks the db to see if we have already
-     * saved this chunk of content before.  If we have, we reuse those same embeddings rather than making a
-     * remote request $$$ to OpenAI for new Embeddings
+     * this method takes a snippet of content and will try to see if we have already generated
+     * embeddings for it. It checks the cache first, and returns if it finds it there.  Then it
+     * checks the db to see if we have already saved this chunk of content before.  If we have, we
+     * reuse those same embeddings rather than making a remote request $$$ to OpenAI for new
+     * Embeddings
+     *
+     * @param content The content that will be tokenized and sent to OpenAI.
```

Comment:
let's be a bit more explicit here with "content" to avoid confusion with dotCMS contents :D

Reply:
Done!